en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,474 @@
+# Get Started with Choreo
+
+Choreo is a full lifecycle cloud-native developer platform that enables your developers to create, deploy and run new digital components like APIs, microservices, and automations in serverless mode on any Kubernetes cluster.
+
+This guide walks you through the following:
+
+- Developing, deploying, testing, and publishing a REST API.
+- How an external application consumes the published REST API.
+- How to release new versions of the REST API with added functionality.
+
+Let's consider a use case where a developer named Jane designs an application for reading lists. A Choreo developer named John develops a REST API for this application to consume. The two developers will develop two versions of the solution in two iterations. The second version of the application will add data persistence as a new feature.
+
+The following diagram summarizes the use case:
+
+![Use case summary](../assets/img/get-started/use-case-summary.png){.cInlineImage-full}
+
+Let's get started!
+
+## Prerequisites
+
+Before you try out this guide, complete the following:
+
+1. Create a GitHub repository to save the REST API implementation. For this guide, you can fork [https://github.com/Asitha/choreo-reading-list-application-example](https://github.com/Asitha/choreo-reading-list-application-example).
+
+2. If you are logging in to Choreo Console for the first time, create an organization as follows:
+
+    1. In [https://wso2.com/choreo/](https://wso2.com/choreo/), click **Try Choreo Now**.
+
+    2. Sign in to Choreo using your Google, GitHub, or Microsoft account.
+
+    3. Enter a unique organization name. For example, `Stark Industries`.
```

Comment:
`Start Industries` was intentional yeah? :D

Reply:
Yes. Used the same example we have been using in other pages that describe the organization creation process like [1].

[1] https://wso2.com/choreo/docs/administer/configure-enterprise-login/